libr/core/cmd_anal.c

```diff
@@ -7499,13 +7499,10 @@ static int cmd_anal(void *data, const char *input) {
 		} else if (input[1] == 't') {
```

Comment:
seems like `abt` is not listed in the help message :( would you mind adding it?

Reply:
done